ios-nocodesign-9-1-armv7.cmake

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2015, Ruslan Baratov
+# All rights reserved.
+
+if(DEFINED POLLY_IOS_NOCODESIGN_ARMV7_CMAKE)
+  return()
+else()
+  set(POLLY_IOS_NOCODESIGN_ARMV7_CMAKE 1)
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_clear_environment_variables.cmake")
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_init.cmake")
+
+set(IOS_SDK_VERSION 9.1)
+set(POLLY_XCODE_COMPILER "clang")
+polly_init(
+    "iOS ${IOS_SDK_VERSION} Universal (iphoneos + iphonesimulator) / \
```

Comment:
`armv7` should be used here instead of `Universal`.

Just for your information I'm planning to generate this string automatically in future.